plugins/woocommerce-admin/client/marketing/overview-multichannel/LearnMarketing.tsx

```diff
@@ -0,0 +1,191 @@
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-title">
+				{ post.title }
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-description">
+				{
+					// translators: %s: author's name.
+					sprintf( __( 'By %s', 'woocommerce' ), post.author_name )
+				}
+				{ post.author_avatar && (
+					<img
+						src={ post.author_avatar.replace( 's=96', 's=32' ) }
+						alt=""
+					/>
+				) }
+			</div>
+		</a>
+	);
+};
+
+const blogPostCategory = 'marketing';
+const perPage = 2;
+
+const LearnMarketing = () => {
+	const [ page, setPage ] = useState( 1 );
+	const { isLoading, error, posts } = useSelect(
+		( select ) => {
+			const { getBlogPosts, getBlogPostsError, isResolving } =
+				select( STORE_KEY );
+
+			return {
+				posts: getBlogPosts( blogPostCategory ),
+				isLoading: isResolving( 'getBlogPosts', [ blogPostCategory ] ),
+				error: getBlogPostsError( blogPostCategory ),
+			};
+		},
+		[ blogPostCategory ]
+	);
+
+	/**
+	 * Renders card footer.
+	 *
+	 * - If loading is in progress, it returns a loading placeholder in the card footer.
+	 * - If there is an error or there are no posts, there will be no card footer.
+	 * - Returns a pagination component in the card footer for paging through the posts.
+	 */
+	const renderFooter = () => {
+		if ( isLoading ) {
+			return (
+				<div className="woocommerce-marketing-learn-marketing-card__footer--placeholder"></div>
+			);
+		}
+
+		if ( error || posts.length === 0 ) {
+			return null;
+		}
+
+		return (
+			<Pagination
+				showPagePicker={ false }
+				showPerPagePicker={ false }
+				page={ page }
+				perPage={ perPage }
+				total={ posts && posts.length }
+				onPageChange={ ( newPage: number ) => {
+					setPage( newPage );
+				} }
+			/>
+		);
+	};
+
+	/**
+	 * Renders card body, which should display two posts in one page.
+	 *
+	 * - If loading is in progress, it returns two placeholder post tiles.
+	 * - If there is an error, it returns an `EmptyContent` component with error message.
+	 * - If there are no posts, it returns an `EmptyContent` component with "No posts yet" message.
+	 * - Else, it returns two post tiles.
+	 */
+	const renderBody = () => {
+		if ( isLoading ) {
+			return (
+				<>
+					<PlaceholderPostTile />
+					<PlaceholderPostTile />
+				</>
+			);
+		}
+
+		if ( error ) {
+			return (
+				<EmptyContent
+					title={ __(
+						"Oops, our posts aren't loading right now",
+						'woocommerce'
+					) }
+					message={ <ReadBlogMessage /> }
+					illustration=""
+					actionLabel=""
+				/>
+			);
+		}
+
+		if ( posts.length === 0 ) {
+			return (
+				<EmptyContent
+					title={ __( 'No posts yet', 'woocommerce' ) }
+					message={ <ReadBlogMessage /> }
+					illustration=""
+					actionLabel=""
+				/>
+			);
+		}
+
+		return (
+			<>
+				<PostTile post={ posts[ ( page - 1 ) * perPage ] } />
+				<PostTile post={ posts[ ( page - 1 ) * perPage + 1 ] } />
```

Comment:
💅  Potential refactor in `( page - 1 ) * perPage`

Reply:
Also I don't get this... I mean what happen if `perPage` is set to 3,4,5,8... ?? it will still show 2 post only?

Reply:
Thanks, good catch, I was developing it based on the visual design and forgot about the `perPage` variable. 😄 I have fixed it in [`99d96db` (#34264)](https://github.com/woocommerce/woocommerce/pull/34264/commits/99d96db714231e642f9cf19a06608cd972f70a8b).